Formatting changes to templates (#1667)
GoogleCloudPlatform/DataflowTemplates: v2/gcs-to-sourcedb/README_GCS_to_Sourcedb.md

* **windowDuration** : The window duration/size in which data is written to Cloud Storage. Allowed formats are: Ns (for seconds, example: 5s), Nm (for minutes, example: 12m), Nh (for hours, example: 2h). If not provided, the value is taken from spanner_to_gcs_metadata. If provided, this takes precedence. To be given when running in regular run mode. (Example: 5m).
* **runMode** : Regular writes to source db, reprocess does processing the specific shards marked as REPROCESS, resumeFailed does reprocess of all shards in error state, resumeSuccess continues processing shards in successful state, resumeAll continues processing all shards irrespective of state. Defaults to: regular.
* **metadataTableSuffix** : Suffix appended to the spanner_to_gcs_metadata and shard_file_create_progress metadata tables.Useful when doing multiple runs.Only alpha numeric and underscores are allowed. Defaults to empty.
* **transformationJarPath** : Custom jar location in Cloud Storage that contains the custom transformation logic for processing records in reverse replication. Defaults to empty.

> * **transformationClassName** : Fully qualified class name having the custom transformation logic.  It is a mandatory field in case transformationJarPath is specified. Defaults to empty.

Reviewer: ```suggestion
* **transformationClassName** : Fully qualified class name for the class that contains the custom transformation logic. When `transformationJarPath` is specified, this field is required. Defaults to empty.
```

> * **transformationCustomParameters** : String containing any custom parameters to be passed to the custom transformation class. Defaults to empty.

Reviewer: ```suggestion
* **transformationCustomParameters** : The string that contains any custom parameters to pass to the custom transformation class. Defaults to empty.
```

> * **writeFilteredEventsToGcs** : This is a flag which if set to true will write filtered events from custom transformation to GCS. Defaults to: false.

Reviewer: ```suggestion
* **writeFilteredEventsToGcs** : When set to `true`, writes filtered events from custom transformation to Cloud Storage. Defaults to `false`.
```
